source/includes/app-services-cli-overview.rst

```diff
@@ -0,0 +1,106 @@
+.. _cli-overview:
```

Comment:
why is this an include now? based on cursory look, seems like it's only used 1 place

Reply:
This will make it easier to regenerate the page